backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1355,6 +1355,9 @@ databaseChangeLog:
       id: populate-specimen-tables
       author: bwarfield@cdc.gov
       comment: Populate specimen tables and relations based on existing data.
+      validCheckSum: # yes there are two of them because liquibase checksums are not platform-independent LOLsob.
```

Comment:
LOL WHAT

Reply:
I KNOW. 🤦

Reply:
From stackoverflow:
> Yes, it depends on content and DB type, because actually checksum depends not on your changeset text, but on sql which is generated for current DB, which will slightly different for different DBs.
> 
> If you want to avoid checksum errors, you can use validCheckSum tag in changesets.

Something to be aware of if something happens to change the sql generated. Not sure if this includes migrating off our own instance of postgresql onto postgresql-as-a-service or even upgrading the db server. I see devs complaining that the checksums changed when they upgraded mssql for example.

There's also this bug when someone upgraded liquibase across major revisions caused the checksums to be invalid: https://github.com/liquibase/liquibase/issues/1192

Reply:
Yeah that's actually the opposite of what's happening here (which makes sense, since the DB version is very carefully the same)—the migrations where they wrote the SQL based on structured input are consistent, but the migrations where we wrote the SQL are changing.

Reply:
Yes. You are correct it is the opposite; however, the _way_ the checksum works could bite us again in the future in a different way. Was wondering if we shouldn't "lock" validCheckSum into past changelists as a best practice? Maybe overkill.